src/components/notification/_inline-notification.scss

```diff
@@ -17,6 +17,7 @@
     @include font-smoothing;
     @include font-size('14');
     display: flex;
+    justify-content: space-between;
```

Comment:
Change: fixes missing space between inline-notification text and close button.

![image](https://cloud.githubusercontent.com/assets/4185382/26649256/f3b1a08a-460a-11e7-9b16-faa1522b2c5a.png)

☝️  missing space